rosbag2/src/rosbag2/sequential_reader.cpp

```diff
@@ -41,11 +41,20 @@ void
 SequentialReader::open(
   const StorageOptions & storage_options, const ConverterOptions & converter_options)
 {
-  storage_ = storage_factory_->open_read_only(storage_options.uri, storage_options.storage_id);
+  file_uri_ = storage_options.uri;
```

Comment:
instead of copying this one by one, can't we just keep a local copy of storage_options? This will scale better as we add more options.